content/participate/test.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: project
+title: Testing
+section: participate
+tags:
+  - outreach-programs
+  - developer
+---
+
+== Contribute
```

Comment:
Not needed? Would be better to have Test Automation and Manual testing as categories, or so